addons/website_blog/views/website_blog_templates.xml

```diff
@@ -25,549 +25,562 @@
+            <div class="oe_structure oe_empty o_wblog_structure_shared"
+                data-editor-message="DRAG BUILDING BLOCKS HERE"
+                data-editor-sub-message="Visible in all blogs' pages"/>
         </div>
     </t>
 </template>
 
-<!-- Blog Post List: Displaying a list of Blog Posts -->
+<!-- ======  Blog(s) Index : Displaying a list of Blog Posts ===================
+Used by 'All blogs' and 'blog' (to share the same layout options) and to display
+list of filtered posts (by date or tag).
+
+Options:
+# opt_blog_cover_post: Replace top-banner content with the latest published post
+    > opt_blog_cover_post_fullwidth_design: use fullwith design
+    > opt_blog_cover_post_show_author: show the author
+
+# opt_blog_cards_design: wrap posts in a standard bts cards components
+# opt_blog_list_view: display post in a list rather than a grid
+# opt_blog_sidebar_show: show sidebar
+# opt_blog_blogs_nav_show: show blogs navigation bar
+# opt_blog_readable: increase font-size, adapt layout
+============================================================================ -->
 <template id="blog_post_short" name="Blog Posts">
     <t t-call="website_blog.index">
-        <t t-set="head">
+        <t t-set="head" t-if="blog">
             <link t-att-href="'/blog/%s/feed' % (blog.id)" type="application/atom+xml" rel="alternate" title="Atom Feed"/>
             <meta t-if="len(active_tag_ids) > 0" name="robots" t-attf-content="#{len(active_tag_ids) > 1 and 'noindex,'} nofollow"/>
         </t>
-        <div class="oe_structure" id="oe_structure_website_blog_post_short_1"/>
-        <div class="container">
-            <t t-call="website.pager" >
-                <t t-set="classname">float-right</t>
+
+        <section id="o_wblog_blog_top">
+            <!-- Selectively display droppable-areas for 'all blogs'/single-blog pages -->
+            <t t-if="not tag and not date_begin">
+                <div id="o_wblog_blog_top_droppable">
+                    <div t-if="blog" t-field="blog.content" class="oe_structure oe_empty" data-editor-message="DRAG BUILDING BLOCKS HERE" t-attf-data-editor-sub-message="Visible on this page only (#{blog.name})." />
+                    <div t-elif="blogs" class="oe_structure oe_empty" data-editor-message="DRAG BUILDING BLOCKS HERE"/>
+                </div>
             </t>
-        </div>
-        <div class="oe_structure">
-            <section class="mb0">
-                <div class="container">
-                    <div class="row">
-                        <div class="col-lg-12 mb32 mt16 text-center">
-                            <h1 t-field="blog.name"/>
-                            <h3 class="text-muted" t-field="blog.subtitle"/>
-                        </div>
+            <!-- Droppable-area for filtered results (tags or date) -->
+            <div t-if="tag or date_begin" class="oe_structure oe_empty" data-editor-message="DRAG BUILDING BLOCKS HERE" contenteditable="true"/>
```

Comment:
ok for all, thanks 👍 

> Why not a simple oe_structure element on top ?

Because each single blog +  the page 'All blogs' header + the page 'Filtered Results' header (by tags or by date)  may have his custom banner. This banners are not mandatory of course, you can use the latest post (or nothing) instead.